modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/FileEntryPicker.js

```diff
@@ -36,7 +37,16 @@ const FileNamePicker = ({
 	}, [inputValue]);
 
 	const onInputChange = ({target}) => {
-		if (target.files[0].size > maxFileSize) {
+		const fileType = target.files[0]?.type;
+		const maxFileTypeSize = Number(maxMimeTypeSize[fileType]);
+		let maxSize = Number(initialMaxFileSize);
+
+		if (maxFileTypeSize && maxSize > maxFileTypeSize) {
+			maxSize = maxFileTypeSize;
+		}
+		setMaxFileSize(maxSize);
```

Comment:
Shouldn't there be a blank line before the call to `setMaxFileSize`?

Reply:
I updated!